packages/app/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "types": ["@emotion/react/types/css-prop"],
+    "types": ["@emotion/react/types/css-prop", "node"],
```

Comment:
Why is this needed?

Reply:
For structuredClone. I think this is included by default with newer versions of node, but if you're on 18(?) you need to add this.